src/dpl/test/dpl_aux.py

```diff
@@ -0,0 +1,91 @@
+import utl
+import re
+
+def detailed_placement(design, *, max_displacement=None, suppress=False):
```

Comment:
What is '*' for?

Reply:
In Python, all args can be by position or by keyword. This * is a relatively new syntax feature. It forces all args after the * to be keyword only, ie no positional, if you want to change the default values. I think this is a lot safer and it kind of matches the tcl interface.

Reply:
Thanks I learned something :)